src/main/scala/io/iohk/ethereum/vm/OpCode.scala

```diff
@@ -834,9 +834,9 @@ sealed abstract class CallOp(code: Int, delta: Int, alpha: Int) extends OpCode(c
   }
 
   private def gasCap[W <: WorldStateProxy[W, S], S <: Storage[S]](state: ProgramState[W, S], g: UInt256, gExtra: UInt256): UInt256 = {
-   if (state.gas >= gExtra)
+    if (state.config.subGasCapDivisor.isDefined && state.gas >= gExtra)
```

Comment:
The problem is that `subGasCapDivisor` carries 2 pieces of information: whether to perform a calculation and how to perform it. If we did both the check and the calculation here, then we would have to repeat it for CREATE. So I don't see a simple solution for that :(